.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
```

Comment:
Neat thing that Rocky introduced to me here. This creates an option to [trigger the workflow manually](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) if needed. Could come in handy.

Reply:
Yep, I also got exposure to this via: https://github.com/azavea/scala-steward/blob/e75d8ffdb3556fed2834af413f6f1162efb88bca/.github/workflows/scala-steward.yml#L3-L6